signus/freedesktop/io.github.signus_game.signus.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.0-->
+  <id>io.github.signus_game.signus</id>
+  <name>Signus: The Artefact Wars</name>
+  <summary>turn-based strategy game</summary>
+  <developer_name></developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <description>
+    <p>Signus is a Czech turn-based strategy game from 1998, developed by Valacirca and released by Zima Software. Since 2002 it is free and open source software with both code and assets.</p>
+    <p>The game features 40 units: scout vehicles, tanks, artillery, helicopters, planes, ships, and support units providing repairs, refuels and ammo. There are also nine buildings and five defense towers in the game.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://signus.paan.cz/wp-content/uploads/2013/03/mise7-1.jpg</image>
```

Comment:
This screenshot shows the original commercial version of the game. The open version uses different fonts.

Reply:
Can you point me to a URL of an alternative?

Reply:
I've tweeted a few screenshots during development:
https://twitter.com/next_ghost_cz/status/1398376572277624842
https://twitter.com/next_ghost_cz/status/1325556150562136064

Or there's a bunch of good screenshots in this article but I don't know how much traffic the hosting can take:
https://www.high-voltage.cz/2021/signus-the-artefact-wars-zapomenuta-klasika-valecnych-tahovek/

Please don't use the repo picture for social media, it's horribly distorted by GitHub.

Reply:
The screenshots will be mirrored. I will take the English language Twitter ones.